python/sglang/test/test_programs.py

```diff
@@ -517,3 +517,35 @@ def few_shot_hellaswag(s, question, choices):
     accuracy = np.mean(np.array(preds) == np.array(labels))
 
     return accuracy, latency
+
+
+def test_gen_min_new_tokens(model_path):
```

Comment:
Use `global_config.default_backend.endpoint.get_model_name()` to get the model name?

Reply:
cool! didn't know we can do that